pulpcore/pulpcore/constants.py

```diff
@@ -36,4 +37,22 @@
     (SYNC_MODES.MIRROR, 'Add new content and remove content is no longer in the remote repository.')
 )
 
+
+POLICY_MODE = SimpleNamespace(
+    IMMEDIATE='immediate',
+    ON_DEMAND='on_demand',
+    CACHE_ONLY='cache_only',
+)
+
+POLICY_CHOICES = (
```

Comment:
Perhaps we should also have these descriptions in the docs somewhere?

Reply:
Yes I agree, and an explanation of the whole lazy thing in general. Maybe I should go ahead and write this page?  https://docs.pulpproject.org/en/3.0/nightly/workflows/deferred-download.html

What about also me renaming that page to "Lazy Download"? Deferred Download just sounds so formal to me.

@asmacdo what do you think? ^

Reply:
Sounds good to me. The only hesitation I have is if this workflow differs plugin to plugin. For now, I think what you propose makes sense, if we get variation, we can move those docs to the plugin_template and each plugin.

Reply:
@asmacdo I wrote this story which I can also do today.  https://pulp.plan.io/issues/4218

Can you help me groom that story? Then I can write the docs in another PR very soon.

Reply:
I'm taking the other PR as assigned so that we can merge this one. Thanks for grooming it @asmacdo